rclpy/test/test_node.py

```diff
@@ -1135,9 +1135,10 @@ def test_describe_parameters(self):
         self.assertEqual(foo_descriptor.integer_range[0].to_value, 10)
         self.assertEqual(foo_descriptor.integer_range[0].step, 2)
 
+        # The descriptor gets the type of the parameter.
         bar_descriptor = descriptor_list[1]
         self.assertEqual(bar_descriptor.name, 'bar')
-        self.assertEqual(bar_descriptor.type, ParameterType.PARAMETER_INTEGER)
+        self.assertEqual(bar_descriptor.type, ParameterType.PARAMETER_STRING)
```

Comment:
Can we instead declare the variable with an integer type so not all of the tests have type string?

Reply:
Done in https://github.com/ros2/rclpy/pull/357/commits/b093de0109f78044588ab109de2857e343b788d4.